modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -15,16 +15,16 @@ Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you mu
 [[supported_versions]]
 == Software Requirements
 
-Your Anypoint Service Mesh installation requires the following applications and Kubernetes environments:
+Your Anypoint Service Mesh installation requires you to:
 
-* One of the following environments on which to run Kubernetes:
+* Have installed one of the following environments on which to run Kubernetes:
```

Comment:
we usually avoid tenses like "have install"

The prerequisite are:
* Install x

Reply:
Done! Thanks!